skype/skype-ps/skype/skype.md

```diff
@@ -2080,6 +2089,9 @@ The following cmdlet references are for Skype for Business and Microsoft Teams.
 ### [Remove-CsTeamsMeetingPolicy](Remove-CsTeamsMeetingPolicy.md)
 {{Manually Enter Remove-CsTeamsMeetingPolicy Description Here}}
 
+### [Remove-CsTeamsMeetingBrandingPolicy](Remove-CsTeamsMeetingBrandingPolicy.md)
+{{Manually Enter Remove-CsTeamsMeetingBrandingPolicy Description Here}}
```

Comment:
You need to copy the sentence from the SYNOPSIS to replace the placeholder text

Reply:
Done, copied short cmdlet description to skype.md